src/views/Playgrounds/PlaygroundChannelEditCreate.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useRef, useState, useEffect } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+
+import { ImageCropDialog, ImageCropDialogImperativeHandle, ViewWrapper } from '@/components'
+import { ChannelCover, FormField, Select, SelectedItem, HeaderTextField, Tooltip } from '@/shared/components'
+
+import { transitions } from '@/shared/theme'
+import {
+  StyledTitleSection,
+  TitleContainer,
+  StyledFormField,
+  StyledTextarea,
+  StyledActionBarTransaction,
+  StyledAvatar,
+  InnerFormContainer,
+} from './PlaygroundChannelEditCreate.style'
+import { Header, SubTitle, SubTitlePlaceholder, TitlePlaceholder } from '../ChannelView/ChannelView.style'
+
+import { requiredValidation, textFieldValidation } from './formValidationOptions'
+import { formatNumberShort } from '@/utils/number'
+
+const languages: SelectedItem[] = [
+  { name: 'English', value: 'english' },
+  { name: 'Polish', value: 'polish' },
+  { name: 'Spanish', value: 'spanish' },
+]
+
+const publicnessItems: SelectedItem[] = [
+  { name: 'Public (Anyone can see this video', value: 'public' },
+  { name: 'Private', value: 'private' },
```

Comment:
Better value here would probably be `Unlisted` since the channel will be always public (it's in the blockchain after all). Also the value should be `true/false`. The corresponding field is `isPublic`